lib/xcsoar/mapgen/server/views/index.html

```diff
@@ -57,7 +57,7 @@
       <div id="map"/>
       <em class="note">Draw the map bounds rectangle by clicking in the map.</em>
       <hr/>
-      <label class="left" for="left">Minimum Longitude:</label><input id="left" type="text" name="left"/><br/>
+      <label class="left" for="left">Minimum Longitude:</label><input id="left" type="text" name="left" value="43"/><br/>
```

Comment:
why the default value=43?

Reply:
I think that snuck in there while I was trying to figure out how to display the waypoint bounds.  I have removed it.